cucumber.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    formatOptions: {
+        snippetInterface: 'async-await',
+    },
+    paths: ['features/*.feature'],
+    dryRun: false,
+    import: [
+        'var/cucumber-init.ts',
+        'features/common/*.ts',
+        'features/steps/*.ts', // this isn't right!
```

Comment:
What's not right?

Reply:
Sorry I didn't see this. The step file should be specific to the feature file but I couldn't see anything in the documentation which allows for doing this. Not an issue at the moment but likely to cause one later on down the road.